sdk/storage/storage-blob/src/utils/constants.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 export const SDK_VERSION: string = "12.1.2";
-export const SERVICE_VERSION: string = "2019-07-07";
+export const SERVICE_VERSION: string = "2019-12-12";
```

Comment:
Lin has 2019-10-10, would better to communicate on the integration and finalize plan

Reply:
Lin will update contants after all PRs merged into stg73 branch